src/commands/gemini.ts

```diff
@@ -0,0 +1,31 @@
+import { cli } from "../cli.js";
+import prettier from "../plugins/prettier.js";
+import { unminify } from "../unminify.js";
+import babel from "../plugins/babel/babel.js";
+import { verbose } from "../verbose.js";
+import { geminiRename } from "../plugins/gemini-rename.js";
+import { env } from "../env.js";
+
+export const azure = cli()
+  .name("gemini")
```

Comment:
From a code maintenance perspective, I wonder if the export `azure` should be changed to `gemini` to align with the name and filename?